commands/http.js

```diff
@@ -160,6 +160,8 @@ module.exports = {
 
     new HTTPRetryRequest(config, reqURL, opts)
       .then(res => {
+        log.debug("res");
+        log.debug(res);
```

Comment:
Small nitpick. Could we make this similar to the other formats? or similar? Otherwise good to go

```
log.debug(` \n Response: ${res}`);
```